### DISCUSSION TITLE
Installation on Mac OS

### DISCUSSION THREAD

**Discussion by digs1998** [DigitalSlideArchive/digital_slide_archive#306] [General]
Hi, I was wondering if there is any information on how to install histomics UI on Mac OS, as the current repo only shows the information on how to set it up on Linux.

**Comment by manthey** [comment]
You can use docker and docker-compose.

**Comment by digs1998** [comment]
I haven't used them before, do you have any resources I can use?

**Comment by manthey** [comment]
You need docker installed.  See https://docs.docker.com/desktop/install/mac-install/.  Once installed, the instructions for docker installation should work: https://github.com/DigitalSlideArchive/digital_slide_archive/blob/master/devops/dsa/README.rst

**Comment by digs1998** [comment]
Thanks, I'll check it out.
